### PR TITLE
docs: describe Skyrim files needed for running tests

### PR DESCRIPTION
**Pull request by nic11**

### CODE REVIEW

**Comment by Pospelove** [comment]
Good. I need to think if we should point it anywhere else.

**Comment by Pospelove** [comment]
We may check CRC32 and file sizes we test with. Of course, generally, SHA512 or something like that would be better, but CRC32 is already used in our project and Skyrim-related tooling.

**Comment by nic11** [comment]
I was also thinking about it, but thought it's an overkill at first. But let's add that, though it will need some changes in espm, I guess (or not)

**Comment by nic11** [comment]
And yep, that will help catching issues with tests failing on modded files, so that will be definitely useful

**Comment by Pospelove** [comment]
Ok, let's create an issue about CRC32, etc.
Maybe we should add these comments to docs instead of source code? Or in both places?

**Comment by nic11** [comment]
You mean creating something like `docs/docs_running_tests.md` and placing it there?

**Comment by Pospelove** [comment]
Not sure about docs so merging as it is

**Comment by Pospelove** [comment]
> You mean creating something like `docs/docs_running_tests.md` and placing it there?

not sure where exactly but I think not in source files

**Comment by nic11** [comment]
That's probably right, but let's keep a reference from tests to docs then

**Comment by Pospelove** [comment]
Yes

**Comment by nic11** [comment]
Made #107, I'll think how to point that out better. I'll also change this PR later. I guess it'll be best to add this as an optional step after building

**Comment by nic11** [comment]
Described unit tests in project root `README.md` for now. We'll probably have to make separate file (or files) describing building and running the project, as well as testing changes.

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Playing on Linux isn't supported currently, but a full-featured server is.
    cd build
    cmake .. -DSKYRIM_DIR="C:/Program Files (x86)/Steam/steamapps/common/Skyrim Special Edition"
    ```
-   For Linux users and users who don't have Skyrim SE installed:
+   For users who don't have Skyrim SE installed:
```
Comment (nic11) [inline README.md:68]: Actually, Skyrim is pretty [playable on Linux](https://www.protondb.com/app/489830) :)

And, technically, one can pass path to game directory from Windows partition when dual-booting. So I don't think this note is necessary.

P.S. I can move this to a separate PR, but I don't think it really makes too much sense to be that much granular.

Comment (Pospelove) [reply README.md:68]: This change is ok for me

**Comment by Pospelove** [comment]
@nic11 I've changed this a bit. Please let me know if you think it's mergeable or not.

**Comment by nic11** [comment]
Looks fancy! I didn't try running with ctest, but everything else looks good. Let's merge

**Comment by Pospelove** [comment]
Btw ctest is used in CI

**Pull request merged**